forwards_paper.tex

```diff
@@ -822,46 +822,42 @@ \subsubsection*{Computational complexity}
 but, how does it perform in theory?
 % The simulation results shown in Figures \ref{fig:runtimes_selection} and \ref{fig:relative_speedup_selection} show that
 % pedigree tracking greatly speeds up forward-time simulations.  Further, the run times with pedigree tracking become
-% near-linear in $\rho$ (Figure \ref{fig:runtimes_selection}).  
-In this section, we explore the how required computational effort and storage space scale
+% near-linear in $\rho$ (Figure \ref{fig:runtimes_selection}).
+In this section, we explore how the required computational effort and storage space scale
 in a Wright-Fisher model with recombination.
 
 %% Memory reduction thanks to simplification
-Consider a simulation of a Wright-Fisher population of $2N$ haploid individuals
+Consider a simulation of a Wright-Fisher population of $N$ haploid individuals
```

Comment:
Changed 2N to N here. We don't care about diploidy and so on, right?

Reply:
Sure, OK.